dictionary_app/app.py

```diff
@@ -0,0 +1,87 @@
+import pymongo
+from flask import Flask, render_template, request, redirect
+import requests
+
+client = pymongo.MongoClient("mongodb+srv://base_user:base_user_password@cluster0.dbcb9.mongodb.net/first")
+db = client.first
+collection = db.gokberk
+client_id = "82ffff3f23534eedba167129f0ea8e31"
+secret = "c8025a5e15754c199d82c249969f484f"
```

Comment:
These lines of code are irrelevant with the project I believe. Discarding unnecessary parts would give clear code.
```suggestion
```